en-US/Language.xml

```diff
@@ -238,7 +251,9 @@
 				<term>dig deeper, delve deeper</term>
 				 <listitem>
 					<para>
-						"Visit the following web link to dig deeper into [subject] ...". Using "dig deeper" may translate awkwardly. Consider rewording: "For detailed information regarding [subject], see [link]."
+						"Visit the following web link to dig deeper into [subject] ...".
+            Using "dig deeper" may translate awkwardly.
```

Comment:
```suggestion
            Using "dig deeper" might translate awkwardly.
```

Reply:
Sheesh. pick pick pick. That's supposed to fall under your bug of "follow our own rules" :stuck_out_tongue:

Reply:
I thought that would fall under the banner of what Vale catches. And if it doesn't catch "may" already, maybe it should?